Makefile

```diff
@@ -369,11 +377,11 @@ install-e2e-chart: build/toolchain/bin/helm$(EXE_EXTENSION)
 		--set grafana.enabled=false \
 		--set jaeger.enabled=false \
 		--set prometheus.enabled=false \
+		--set openmatch.monitoring.stackdriver.enabled=false \
```

Comment:
Why false? This is going to make it impossible to debug CI failure that happen in the cluster for CI tests.

Reply:
I think there might be a bug when setting the stackdriver. I was able to see the container and pod logs from stackdriver even though I turned it off at here.